astrokat/test/test_scans.py

```diff
@@ -52,6 +52,14 @@ def test_scan_basic_sim(self):
         self.assertIn("Initialising Scan target scan_1934-638 for 30.0 sec", result)
         self.assertIn("scan_1934-638 observed for 60.0 sec", result)
 
+    def test_reference_pointing_scan_basic_sim(self):
+        """Not much to do: check scan initiate log msg"""
+        execute_observe_main("test_scans/reference-pointing-scan-test.yaml")
+        # get result and make sure everything ran properly
+        result = LoggedTelescope.user_logger_stream.getvalue()
+        self.assertIn("Initialising Scan target scan_1934-638 for 30.0 sec", result)
```

Comment:
yaml configuration does not seem to align with the scan durations that lines 60 and 61 are looking for. I wouldn't expect this to pass as is.